src/main/java/uk/gov/companieshouse/scanupondemand/orders/api/model/Item.java

```diff
@@ -0,0 +1,178 @@
+package uk.gov.companieshouse.scanupondemand.orders.api.model;
+
+import com.google.gson.Gson;
+
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.annotation.Id;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+/**
+ * An instance of this represents an item of any type.
+ */
+public class Item {
+
+	@Transient
+	public static final String SEQUENCE_NAME = "items_sequence";
```

Comment:
I don't think we're using these sequences any more?

Reply:
Looks like we do use them at least in the Certificates API which I used as an example

Reply:
Yeh it's no longer required, so it can be removed. We originally used items_sequence to generate and id using the database. However as Vaughan said it has been replaced be IdGeneratorService.